ceph-salt-formula/salt/ceph-salt/cephtools.sls

```diff
@@ -0,0 +1,28 @@
+{% import 'macros.yml' as macros %}
+
+{{ macros.begin_stage('Ceph tools') }}
+
+{{ macros.begin_step('Install cephadm and other packages') }}
+
+install cephadm:
+  pkg.installed:
+    - pkgs:
+        - cephadm
+{% if 'mon' in grains['ceph-salt']['roles'] %}
+        - ceph-common
```

Comment:
```suggestion
        - ceph-common
        - podman
```
? as this is no longer a `Depends`, but a `Recommends`?

Reply:
see https://github.com/SUSE/ceph-bootstrap/pull/78